scripts/demo/run_sample_benchmark_app.bat

```diff
@@ -90,38 +89,19 @@ if not "%python_ver%"=="okay" (
     goto error
 )
 
-:: install yaml python modules required for downloader.py
-if exist "%VENV_DIR%" (
-    echo.
-    echo ###############^|^| Using the existing python virtual environment ^|^|###############
-    echo.
-) else (
-    echo.
-    echo ###############^|^| Creating the python virtual environment ^|^|###############
-    echo.
-    python -m venv "%VENV_DIR%"
-)
-
-call "%VENV_DIR%\Scripts\activate.bat"
-python -m pip install -U pip
-python -m pip install -r "%ROOT_DIR%..\open_model_zoo\tools\downloader\requirements.in"
-
-if ERRORLEVEL 1 GOTO errorHandling
-
-set downloader_dir=%INTEL_OPENVINO_DIR%\extras\open_model_zoo\tools\downloader
-
 for /F "tokens=* usebackq" %%d in (
-    `python "%downloader_dir%\info_dumper.py" --name "%model_name%" ^|
+    `omz_info_dumper --name "%model_name%" ^|
```

Comment:
As @vladimir-dudnik requested, you should print a suggestion how to fix an error if omz tools are not found. Since there is no documentation for now, I think you should just advice to install openvino-dev with frameworks required by current script

Reply:
You should check all tools used to be installed. It is possible one of them is removed manually